packages/core/core/src/requests/WriteBundlesRequest.js

```diff
@@ -92,7 +92,10 @@ async function run({input, api, farm, options}: RunInput) {
           bundleGraphReference: ref,
           optionsRef,
         });
-        let info = await api.runRequest(request);
+
+        // TODO remove?
+        // force to ensure that loadConfig runs for all packagers/optimizers
+        let info = await api.runRequest(request, {force: true});
```

Comment:
Do we need some `invalidateOnEveryBuild` as opposed to `invalidateOnStartup`? (`force` behaves like the hypothetical `invalidateOnEveryBuild`)